brave/browser-laptop
# Update guids by personal-data-changed

fix #4349
requires https://github.com/brave/electron/pull/70

Auditors: @bridiver

Test Plan:
1. Prepare a fresh install brave (version > 0.12.3)
2. Lauch brave
3. open about:autofill
4. Add an address
5. The address entry should show on the page
6. Add a credit card
7. The credit card entry should show on the page
8. Verify address and credit card on http://www.roboform.com/filling-test-all-fields
